notesapi/v1/management/commands/bulk_create_notes.py

```diff
@@ -0,0 +1,69 @@
+import os
+import random
+import uuid
+
+from django.core.management.base import BaseCommand, CommandError
+from django.db import transaction
+
+from notesapi.v1.models import Note
+
+
+class Command(BaseCommand):
+    args = '<total_notes> <notes_per_user> <course_id course_id ...>'
+    help = 'Add N random notes to the database'
+
+    def handle(self, *args, **options):
+        if len(args) < 3:
+            raise CommandError("generate_random_notes takes the following arguments: " + self.args)
+
+        total_notes = int(args[0])
```

Comment:
I've seen the wiki page. At the time it seemed like all inputs were arguments more than options. I'll refactor `notes_per_user` and `course_ids`  to be options with sane defaultssss.